RetailCoder.VBE/UI/Command/Refactorings/RefactorImplementInterfaceCommand.cs

```diff
@@ -24,6 +24,7 @@ public RefactorImplementInterfaceCommand(VBE vbe, RubberduckParserState state, I
 
         public override bool CanExecute(object parameter)
         {
+            return true;
```

Comment:
not good. *Implement Interface* shouldn't be enabled when the parser state isn't `Ready`, otherwise Rubberduck cannot reliably determine what members *are* and what members *aren't* implemented in the module.